linux/LinuxProcessList.c

```diff
@@ -189,6 +189,20 @@ static void LinuxProcessList_updateCPUcount(ProcessList* super, FILE* stream) {
    }
 }
 
+static void LinuxProcessList_updateTime(LinuxProcessList* this) {
```

Comment:
How often would this function be called?

Reply:
Usually once before each process scan. In normal operation on bare OS at most about 10 times a second (minimum interval being `.1` seconds) …